emop-core/src/main/java/edu/cornell/emop/util/Util.java

```diff
@@ -47,4 +50,54 @@ public static void replaceFileInJar(String jarPath, String oldPath, String newPa
             ex.printStackTrace();
         }
     }
+
+    /**
+     * Recursive routine accumulating the set of package names within the project.
+     * Borrowed from DSI.
+     *
+     * @param currRoot the current location
+     * @param dirName the name of the directory
+     * @return set of all package names within the project.
+     */
+    private static Set<String> classFilesWalk(File currRoot, String dirName) {
+        Set<String> packageNameSet = new HashSet<>();
+        File[] files = currRoot.listFiles();
+        if (files == null) {
+            return packageNameSet;
+        }
+        // we want to list all the potential *.class files in this directory
+        File[] classFiles = currRoot.listFiles(new FileFilter() {
+            @Override
+            public boolean accept(File file) {
+                return !file.isDirectory() && (file.getName().endsWith(".class"));
+            }
+        });
+        if (classFiles.length > 0) {
+            // we found a class file, which means we are in a directory with class files (package)
+            // No need to traverse further because AspectJ within(${PACKAGE_NAME}..*) syntax instruments subpackages.
```

Comment:
Is this mention of ASPECTJ a holdover from DSI or a reference to the ASPECTJ usage in DSI

Reply:
I changed this method around so much that it's not really the same as what I had for the DSI plugin. This is specific to eMOP.

Reply:
Then just remove the DSI reference to avoid confusion.